.github/workflows/dockerhub.yml

```diff
@@ -8,9 +8,19 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-    - name: trigger
-      run: curl -X POST https://hub.docker.com/api/build/v1/source/1813a660-2ee5-4583-a238-dd54e9a6ebac/trigger/c8cd9f1f-f269-45bc-9750-a08327257f62/call/
-    
-    
-
-
+      - name: checkout code
+        uses: actions/checkout@v2
+      - name: install buildx
+        id: buildx
+        uses: crazy-max/ghaction-docker-buildx@v1
+        with:
+          version: latest
+      - name: login to docker hub
+        run: |
+          echo "${{ secrets.DOCKER_PASSWORD }}" | docker login -u "${{ secrets.DOCKER_USERNAME }}" --password-stdin
+      - name: build the image
+        run: |
+          docker buildx build \
+            --push \
+            --tag neilpang/acme.sh:latest \

```

Comment:
is it possible to change the tag "latest" ?
If this is the master branch, we use "latest",  otherwise, we use the branch name("dev").

Reply:
I've now added that by having 2 separate build steps, one for "master" and one for "dev": the only difference between them is the label used in Docker that follows your specification!

Reply:
I think GITHUB_REF env can do this job.
https://stackoverflow.com/questions/58033366/how-to-get-current-branch-within-github-actions

https://docs.github.com/en/actions/configuring-and-managing-workflows/using-environment-variables#default-environment-variables

Reply:
Indeed it can, however as you said you want "dev" -> "dev" and "master" -> "latest" so this was just an easier way to achive it!

If you prefer to do it via bash shell before, that can be achieved also, only reason I picked this is that it was easier and totally compounded (it guarantees that won't deploy any other branch than "dev" and "master")

Reply:
@Neilpang as it stands, this PR will work for "latest" and "dev" tagging of the Docker image, but if you want I can make the changes necessary to use the bash style setting of the tag instead... what do you say?